meetings/2017-09-13.md

```diff
@@ -0,0 +1,151 @@
+# Node Foundation TSC Meeting 2017-09-13
+## Links
+
+* **Recording**: * **GitHub Issue**: [TSC#338](https://github.com/nodejs/TSC/issues/388)
+* **Minutes Google Doc**: https://docs.google.com/document/d/1tOeEpEz2Qf7ybjsoFXRrMSpNrEcKIIfnc3eqDYeP0zU/edit?usp=sharing
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1al5l33g9wlNS3fMQHNdpxyhWPoJhLAMtu4ChaAiCmYs>_
+
+## Present
+
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* Anna Henningsen @addaleax (TSC)
+* Brian White @mscdex (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Evan Lucas @evanlucas (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TS
+* James M Snell @jasnell (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Michaël Zasso @targos (TSC)
+* Rich Trott @Trott (TSC)
+* Rod Vagg @rvagg (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* William Kapke @williamkapke (Observer)
+
+## Agenda
+
+* announcements
+
+* review previous meeting
+
+### nodejs/TSC
+
+* New TSC Director/Chair Elections [#326](https://github.com/nodejs/TSC/issues/326)
+* TSC Meeting Schedule [#336](https://github.com/nodejs/TSC/issues/336)
+* 2FA for the entire org [#301](https://github.com/nodejs/TSC/issues/301)
+* Candidate "areas" for TSC and Community Committee  [#278](https://github.com/nodejs/TSC/issues/278)
+* Copyright Date [#195](https://github.com/nodejs/TSC/issues/195)
+* Regular roll-up reporting for active working groups [#109](https://github.com/nodejs/TSC/issues/109)
+
+### nodejs/node
+
+* stream: remove {writeableState/readableState}.length [#12857](https://github.com/nodejs/node/pull/12857)
+* Cleanup `_writableState` and `_readableState` access across codebase [#445](https://github.com/nodejs/node/issues/445)
+
+### nodejs/CTC
+
+* Requesting Rod to step down or be removed from the CTC [#165](https://github.com/nodejs/CTC/issues/165)
+
+## Announcements
+
+No announcements this week.
+
+## Previous Meeting Review
+
+We skipped this as we had a packed agenda.
+
+## Minutes
+
+### nodejs/TSC
+* New TSC Director/Chair Elections [#326](https://github.com/nodejs/TSC/issues/326)
+ * No objections, Myles acclaimed as Directory, Michael acclaimed as Chair
```

Comment:
Director

Reply:
Would it be better to have it as Michael Dawson, as we had Michaël and Michael in the meeting today?

Reply:
`acclaimed` is the wrong verb here. (It means praised.) I think `named` would be a better word choice. @mhdawson Would you prefer we push nits like that ourselves, or do you prefer to handle them so you can evaluate them?

Reply:
Full names or else GitHub handles seem like the way to go to avoid ambiguity.

Reply:
I'm ok with people pushing the nits themselves.

Reply:
fixed